zerver/openapi/zulip.yaml

```diff
@@ -20291,6 +20303,26 @@ paths:
                   example: false
                 message_retention_days:
                   $ref: "#/components/schemas/MessageRetentionDays"
+                can_add_subscribers_group:
+                  allOf:
+                    - description: |
+                        The set of users who have permission to subscribe others to this
+                        channel expressed as an [update to a group-setting value][update-group-setting].
```

Comment:
This link is broken, because you didn't define `[update-group-setting]`. I'll fix that here and in 2-3 previous ones with the same bug, but a cardinal rule of documentation is that the links almost never work if you don't test clicking them :).